addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -936,9 +937,12 @@ export class Rtc extends Record {
     }
 
     /**
-     * @param {import("models").Thread}
+     * @param {import("models").Thread} channel
+     * @param {object} [options]
+     * @param {boolean} [options.audio]
+     * @param {boolean} [options.video]
      */
-    async joinCall(channel, { video = false } = {}) {
+    async joinCall(channel, { audio = true, video = false } = {}) {
```

Comment:
Is audio both ways or only mute mic? The name might be misleading depending on the answer.

Reply:
It is the audio from the client's input source (usually micrphone), although in hindsight I think that they should be consistent with the other ["stream types"](https://github.com/odoo/sfu/blob/master/src/client.js#L41), and rename `video` into `camera`, if it brings confusion, I can either lengthen their names to indicate that it refers to the client's (input) media, or indicate it in the docstring (like naming the object parameter).